### DISCUSSION TITLE
Partnet-mobility dataset with mujoco / bullet

### DISCUSSION THREAD

**Discussion by shivanimall** [haosulab/SAPIEN#77] [Q&A] [answered]
Hello, 

Is it possible to use partnet-mobility dataset with robosuite / mujoco? Is there any integration pipeline that you may have?

Thanks for your help.

**Comment by fbxiang** [comment, marked as answer]
The dataset uses the URDF format, which should be supported by bullet and probably mujoco as well. I know a lot of people using our dataset in bullet just fine.

**Comment by johnny-wang16** [comment]
I tried loading the URDF directly on mujoco and it doesnt work out of the box. The error message says: "Error: faces of mesh xx have inconsistent orientation. Please check the faces containing the vertices 1 Object nmea = xx, id =xx".
